src/wolfgame.py

```diff
@@ -6578,6 +6604,10 @@ def transition_night(cli):
                 pm(cli, chump, messages["seer_turn"])
                 newrole = "doomsayer"
                 debuglog("{0} ({1}) TURNED DOOMSAYER".format(chump, chumprole))
+            elif chumprole in var.TOTEM_ORDER and chumprole != "wolf shaman": # just in case
```

Comment:
extra check not needed, wolf shamans will never get to this point per the check on 6589